app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,24 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order': order.identifier
+          }
+        };
+        let resend = await this.loader.post('orders/resend-email', payload);
+        if (resend.status) {
```

Comment:
Comment regarding this added on the server PR. https://github.com/fossasia/open-event-server/pull/6163#discussion_r302045104